cmd/renterd/main.go

```diff
@@ -74,18 +76,30 @@ var (
 		},
 		ShutdownTimeout: 5 * time.Minute,
 		Database: config.Database{
-			Log: config.DatabaseLog{
-				IgnoreRecordNotFoundError: true,
-				SlowThreshold:             100 * time.Millisecond,
-			},
 			MySQL: config.MySQL{
 				Database:        "renterd",
 				User:            "renterd",
 				MetricsDatabase: "renterd_metrics",
 			},
 		},
 		Log: config.Log{
-			Level: "warn",
+			Path:  "", // deprecated. included for compatibility.
+			Level: "info",
+			File: config.LogFile{
+				Enabled: true,
+				Format:  "json",
+				Path:    os.Getenv("RENTERD_LOG_FILE"),
```

Comment:
should this be `RENTERD_LOG_FILE_PATH`? edit: this should probably default to `renterd.log`

Reply:
The name is correct. The `RENTERD_LOG_FILE_PATH` is the deprecated env var and this is the new one that matches hostd's `HOSTD_LOG_FILE`.
Setting it would change the behaviour for people who are using `RENTERD_LOG_FILE_PATH` right now since it overwrites that value.